src/plugins/auto/common.hpp

```diff
@@ -69,6 +69,15 @@ struct DeviceInformation {
     std::string defaultDeviceID;
     DeviceName uniqueName;
     unsigned int devicePriority;
+    DeviceInformation () : numRequestsPerDevices(-1), devicePriority(0) {}
+    DeviceInformation (DeviceName dn, std::map<std::string, std::string> conf, int nReq, std::string defaultID, DeviceName uName, unsigned int priority) {
```

Comment:
Where is the second constructor applied?

Reply:
https://github.com/openvinotoolkit/openvino/blob/06cacfe2a70d6ff5/src/tests/unit/auto/exec_network_get_metrics.cpp#L251
https://github.com/openvinotoolkit/openvino/blob/06cacfe2a70/src/plugins/auto/plugin.cpp#L244
Actually, the parameter number can be from 0 and up to 6, so I am adding another commit.